debian/libmercury1.dirs

```diff
@@ -1 +1,2 @@
+usr/bin
```

Comment:
This should go into a `mercury.dirs` file.  The `libmercury1` package is for the shared library only.

Reply:
ok thanks! We have a similar issue with the libfabric package btw, I was copying what was done there.

Reply:
We should try to fix that on the next round of updates there.  I am doing one for EL9 so I'll take a look at it then.  Or if you do an update there before I can land mine, please do fix that.